### ISSUE TITLE
Make target `latex-doc` broken due to comment in SetValuedFunctors

### ISSUE THREAD

**Issue by benediktahrens** [UniMath/UniMath#558]
@amblafont has broken 
```
$ make latex-doc
```
by putting weird stuff in Coq comments. 

```
\T1/cmr/bx/n/24.88 Math.CategoryTheory.SetValuedFunctors[]  
[1300] [1301]
Runaway argument?
{X a family of equivalence relations on RX satisfying if x \~{}\textit \ETC.
! TeX capacity exceeded, sorry [main memory size=5000000].
l.163534 \coqdocindent{4.00e
                            m}
!  ==> Fatal error occurred, no output PDF file produced!
```

**Comment by benediktahrens** [comment]
On https://coq.inria.fr/refman/Reference-Manual017.html , in the section 'Escaping to LATEX and HTML.' is described how to put latex code and html code in comments.
Be aware that latex code is discarded in HTML doc and vice versa, so both latex and HTML code have to be given to produce sensible output in both docs.

**Comment by benediktahrens** [comment]
Resolved in #697 .
